tests/test_config_consistency.py

```diff
@@ -791,6 +791,15 @@ def parse_log_injection_message(log_message):
                         continue
                     dd_pairs = re.findall(r"dd\.\w+=\S+", match.group(0))  # Extract key-value pairs that start with dd.
                     return {pair.split("=")[0]: pair.split("=")[1] for pair in dd_pairs}
+            elif context.library == "php":
+                match = re.search(r"\[[^\]]+\]\s+\w+\.\w+:\s+.*?({.*?})\s*\[\]", log)
+                if match:
+                    try:
+                        message = json.loads(match.group(1))
+                        if message.get("message") == log_message:
+                            return message
+                    except json.JSONDecodeError:
+                        continue
```

Comment:
```suggestion
                log = match.group(1)
```

Can you add `message: message` to the k/v map pair [here](https://github.com/DataDog/system-tests/blob/4e878cf4be8c29b886965e6df5134cc42d072770/tests/test_config_consistency.py#L34-L39)? That way can just set `log = match.group(1)` and then reuse the rest of the code below rather than duplicating.

Reply:
Yup, I cleaned this up a bit more and moved all the matching logic in the try-except.